know/rdf/tests/format_wikidata_tests.py

```diff
@@ -257,3 +263,1269 @@ def test_format_country(self):
 
         # assert
         self.assertEqual(result, expected)
+    def test_format_person(self):
+        """ Tests format_person """
+        json_input = {

```

Comment:
yeah uh this is 1000 lines, def put this in its own file - I gave you the function for it all you have to do is call it

Reply:
I used a shorter test case